README.md

```diff
@@ -54,12 +53,49 @@ jobs:
           # This is a Personal Access Token and it needs to have the following permissions
           # - "read:org": used to read the project's board
           # - "write:org": used to assign issues to the project's board
-          PROJECT_TOKEN: ${{ steps.generate_token.outputs.token }}
+          PROJECT_TOKEN: ${{ secrets.PROJECT_TOKEN }}
           # The number of the project which the issues will be synced to
           # You can find this in https://github.com/orgs/@ORGANIZATION/projects/<NUMBER>
           project: 4
+          # Optional, the project field to modify with a new value
+          # Found more in https://docs.github.com/en/issues/planning-and-tracking-with-projects/understanding-fields/about-single-select-fields
+          project_field: Status
+          # Optional unless that project_field was set up. Then this field is required.
+          # The value to modify in the project field
+          project_value: To do
+          # Optional, labels to work with.
+          # If this value is set, the system will ignore any issue that doesn't contain that label(s).
```

Comment:
nit: 
sorry, may be that's me, i had to re-read this description to get it right few times :) sounds like there's a double negative structure "ignore" / "doesn't contain"
could it be put like "If label(s) provided, action will apply project fields to only issues with such labels"

Reply:
Makes sense! I'll reworded it before merging!